samples/bluetooth/mesh_demo/src/microbit.c

```diff
@@ -6,6 +6,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <stdbool.h>
```

Comment:
For completeness this is is fine here, but it shouldn't be necessary since `<drivers/gpio.h>` uses `bool`.  Did you encounter problems?

Reply:
> Did you encounter problems?

No; I'll remove it.